src/index.html

```diff
@@ -7,7 +7,232 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header__page">
+        <nav class="nav">
+          <ul class="nav__menu-icons">
+            <li class="nav__icons">
+              <input
+                type="checkbox"
+                id="nav-toggler"
+                class="nav__toggler"
+              >
+              <label class="nav__toggler-label" for="nav-toggler">
+              </label>
+            </li>
+            <li class="nav__icons">
+              <a href="#" class="nav__link-logo"></a>
+            </li>
+            <li class="nav__icons">
+              <a href="tel:+1234555-55-55" class="nav__link-contact"></a>
+              <div class="nav__number-phone">+1 234 555-55-55</div>
+            </li>
+          </ul>
+          <div class="nav__menu-full">
+            <div class="nav__menu-container">
+              <ul class="fullscreen-menu">
+                <li class="fullscreen-menu-item"><a class="fullscreen-menu-link" href="#">home</a></li>
```

Comment:
```suggestion
                <li class="fullscreen-menu-item">
                  <a class="fullscreen-menu-link" href="#">home</a>
                </li>
```
Аналогично и с другими